styles.css

```diff
@@ -472,6 +476,79 @@ h1::after {
   margin-top: 10px;
 }
 
+.contact-container {
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+  padding-bottom: 2rem;
+  margin-top: 4578px;
+}
+
+.contact-option-lists {
+  width: 327px;
+  height: 216px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 54px;
+  margin-top: 73px;
+}
+
+.contact-form li {
+  list-style: none;
+}
+
+input {
+  width: 295px;
+  height: 18px;
+  padding-top: 1.5rem;
+  padding-bottom: 1.5rem;
```

Comment:
- [ ] You did great adding a padding to the top and bottom of the input fields. But the design also has horizontal paddings. Please add padding to the left and right of the text fields.
- [ ] Please remove the border that appears when the text fields are active. Because there is no such kind of border in the Figma design.